projects/js-packages/components/components/alert/index.tsx

```diff
@@ -21,7 +20,17 @@ export const LEVEL_SUCCESS = 'success';
 
 export const ALERT_LEVELS = [ LEVEL_ERROR, LEVEL_WARNING, LEVEL_INFO, LEVEL_SUCCESS ];
 
-const getIconByLevel = level => {
+type AlertProps = {
+	/** The severity of the alert. */
+	level: typeof LEVEL_ERROR | typeof LEVEL_WARNING | typeof LEVEL_INFO | typeof LEVEL_SUCCESS;
+
+	/** Children to be rendered inside the alert. */
+	children: React.ReactElement | string;
```

Comment:
Better to use `ReactNode` here which means a `ReactFragment`, a `string`, a `number` or an array of `ReactNode`, or `null`, or `undefined`, or a `boolean`

```suggestion
	children: React.ReactNode;
```